### PR TITLE
Search

### PR DESCRIPTION
**Pull request by devidsdt**
Reopened the PR, I forgot to rebase the changes in main branch.

### CODE REVIEW

File: src/TuDa.CIMS.Web/Components/AssetSearch.razor [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        <i class="bi bi-search position-absolute"
+           style="top: 50%; right: 15px; transform: translateY(-50%); color: #aaa;">
+        </i>
+    </button>
+</div>
+
+@* Link to Bootstrap Icons *@
+<link
+    href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.0/font/bootstrap-icons.css"
+    rel="stylesheet"/>
```
Comment (Sc3l3t0n) [inline src/TuDa.CIMS.Web/Components/AssetSearch.razor:22]: :thinking: Is this really necessary? I thought the icons are included in the normal package

File: src/TuDa.CIMS.Web/Components/AssetSearch.razor [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+           id="prototypeSearchBar"
+           placeholder="Suche">
+
+    <!-- Search button -->
+    <button type="button"
+            class="position-absolute end-0 top-50 translate-middle-y me-2 btn p-0 border-0 bg-transparent">
+
+        <!-- Search icon -->
+        <i class="bi bi-search position-absolute"
+           style="top: 50%; right: 15px; transform: translateY(-50%); color: #aaa;">
```
Comment (Sc3l3t0n) [inline src/TuDa.CIMS.Web/Components/AssetSearch.razor:14]: :wrench: Can we use bootstrap classes for that, or extract this to `AssetSearch.razor.css` as a new class?

File: src/TuDa.CIMS.Web/Components/AssetSearch.razor [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿@* Search bar for assets *@
+<div class="position-relative"
+     style="max-width: 600px;">
+    <input class="form-control rounded-pill"
+           id="prototypeSearchBar"
```
Comment (Sc3l3t0n) [inline src/TuDa.CIMS.Web/Components/AssetSearch.razor:5]: :thinking: Do you need this?

**Review by Sc3l3t0n — CHANGES_REQUESTED** [review]

File: src/TuDa.CIMS.Web/Components/AssetSearch.razor (hunk 1 above)
Comment (Sc3l3t0n) [reply src/TuDa.CIMS.Web/Components/AssetSearch.razor:22]: Ok, it actually is.

Comment (Sc3l3t0n) [reply src/TuDa.CIMS.Web/Components/AssetSearch.razor:22]: :wrench: You can also just move this to `App.razor`, but then add a TODO comment above it.

**Comment by Sc3l3t0n** [comment]
For what are the `woff` files needed?

File: src/TuDa.CIMS.Web/Components/AssetSearch.razor [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <input class="form-control rounded-pill"
+           placeholder="Suche">
+
+    <!-- Search button -->
+    <button type="button"
+            class="position-absolute end-0 top-50 translate-middle-y me-2 btn p-0 border-0 bg-transparent">
+
+        <!-- Search icon -->
+        <span
+            class="bi bi-search position-absolute top-50 end-0 translate-middle-y text-secondary">
```
Comment (Sc3l3t0n) [inline src/TuDa.CIMS.Web/Components/AssetSearch.razor:13]: ⛏️ Could be converted to a single line

**Review by Sc3l3t0n — CHANGES_REQUESTED** [review]

**Comment by devidsdt** [comment]
> For are the `woff` files needed for?

Otherwise Blazor does not find the icons

**Review by Sc3l3t0n — APPROVED** [review]

**Review by anngy12 — APPROVED** [review]

**Pull request merged**